docs/dev/install.rst

```diff
@@ -53,6 +53,15 @@ Installation
 
       ./bin/peep.py install -r requirements.txt
 
+   .. note::
+  
+     In case you encounter a locale issue while installation, use the following
```

Comment:
As @jotes mentioned, being specific about the error would be helpful here. Something like "You may encounter an error along the lines of `ValueError: blah` while running this command."

You may also want to include a link to the open Python issue: https://bugs.python.org/issue18378

Also, the fix itself needs to be explicit that it's asking the user to add those two lines to their `.bash_profile` or equivalent file.